src/pages/registration/DescriptionPanel.tsx

```diff
@@ -225,7 +223,16 @@ export const DescriptionPanel = () => {
           )}
         </Field>
       </Box>
-
+      {registrationsHaveSamePublicationDate(
+        duplicateRegistration?.entityDescription?.publicationDate,
+        values.entityDescription?.publicationDate
+      ) && (
+        <DuplicateWarning
+          name={duplicateRegistration!.entityDescription?.mainTitle}
+          id={duplicateRegistration!.identifier}
```

Comment:
Siden alt dette forutsetter at man har en duplikat, er det kanskje naturlig å bare legge inn en eksplisitt sjekk på det først, og slipper unna litt `?`/`!` etterpå?
```suggestion
      {duplicateRegistration && registrationsHaveSamePublicationDate(
        duplicateRegistration.entityDescription?.publicationDate,
        values.entityDescription?.publicationDate
      ) && (
        <DuplicateWarning
          name={duplicateRegistration.entityDescription?.mainTitle}
          id={duplicateRegistration.identifier}
```